guides/client/bindings.md

```diff
@@ -436,7 +436,6 @@ Our `paginate_posts` function fetches a page of posts, and determines if the use
   phx-update="stream"
   phx-viewport-top={@page > 1 && "prev-page"}
   phx-viewport-bottom={!@end_of_timeline? && "next-page"}
```

Comment:
Since we used phx-page-loading here, we probably should do

```suggestion
  phx-viewport-top={@page > 1 && JS.push("prev-page", page_loading: true)}
  phx-viewport-bottom={!@end_of_timeline? && JS.push("next-page", page_loading: true)}
```